src/lib/util.js

```diff
@@ -0,0 +1,45 @@
+const chalk = require(`chalk`);
```

Comment:
When I have a `parent.js` that implements features `A`, `B`, and `C` and the functions that actually make those happen are extracted into their own smaller modules, I like to put them in a child directory to represent that. We could name the directory something else, if you feel like `lib/` is not explicit enough. I can see a case for moving `util.js` up into the `src/` directory, alongside `fastly.js`, but I would keep all the extracted modules (`purge-all.js`, `purge.js`, etc.) in a subdirectory to help make things more organized.